src/tsx/layout/Header.tsx

```diff
@@ -29,7 +29,7 @@ function Header() {
     }, [open]);
 
     return (
-        <header className={`fixed w-screen inset-x-0 top-0 z-50 bg-white/75 before:w-screen before:transition-all before:ease-in-out before:duration-300 before:h-screen before:absolute before:bg-grey-900 before:-z-10 lg:before:transition-none ${open ? 'before:visible before:opacity-60' : 'before:opacity-0 before:invisible'}`}>
```

Comment:
Is this a change in indentation? It looks a bit off to me

![image](https://github.com/SmartCityFlensburg/project-website/assets/73289312/74c0fbd3-b037-4fe4-9419-195ad71a14b2)

Reply:
Are we talking about the same thing? I'm talking about the header. You removed `bg-white/75` from the header, before it looked like this, so the text from the header isn't inside the text on the page

![image](https://github.com/SmartCityFlensburg/project-website/assets/73289312/9ac611cf-b14d-4847-82e4-7a0e40b8e67c)